snippets/olmv1-multi-catalog-admon.adoc

```diff
@@ -0,0 +1,13 @@
+// Text snippet included in the following modules:
+//
+// * modules/olmv1-about-catalogs.adoc
+
+:_mod-docs-content-type: SNIPPET
+
+[IMPORTANT]
+====
+If you try to install an Operator or extension that does not have unique name, the installation might fail or lead to an unpredictable result. This occurs for the following reasons:
+
+* If mulitple catalogs are installed on a cluster, {olmv1} does not include a mechanism to specify a catalog when you install an Operator or extension.
+* Dependency resolution in {olmv1-first} assumes that all of the Operators and extensions that are available to install on a cluster use a unique name for their bundles and packages.
```

Comment:
_assumes_ seems anthropomorphic to me. ISG identifies: _decide_, _think_, _want_ as anthropomorphic.

```suggestion
* Dependency resolution in {olmv1-first} requires/ensures that all of the Operators and extensions that are available to install on a cluster use a unique name for their bundles and packages.
```